scala/wdl_scala/project/plugins.sbt

```diff
@@ -0,0 +1,9 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.6.4")
+
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.3")
+
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.0")
```

Comment:
Bump this to `0.14.1` and delete the file `scala/wdl_scala/project/assembly.sbt`